tools/test-proxy/Azure.Sdk.Tools.TestProxy/RecordingHandler.cs

```diff
@@ -105,10 +116,10 @@ public void StopRecording(string id, bool saveToDisk)
             }
         }
 
-        public void StartRecording(string fileId, HttpResponse outgoingResponse)
+        public void StartRecording(string sessionId, HttpResponse outgoingResponse)
         {
             var id = Guid.NewGuid().ToString();
-            var session = (fileId, new ModifiableRecordSession(new RecordSession()));
+            var session = (sessionId??"", new ModifiableRecordSession(new RecordSession()));
```

Comment:
```suggestion
            var session = (sessionId ?? String.Empty, new ModifiableRecordSession(new RecordSession()));
```

NIT: Spaces before and after `??`, and in .NET you should always use `String.Empty` instead of `""`.